site/pages/our-work/case-study-fortnum-and-mason-digital-transformation-slice/style.css

```diff
@@ -0,0 +1,95 @@
+@value badgerBlack, badgerWhite, badgerRedOnWhite, linesOnWhite from "../../../css/_colors.css";
+@value mediumScreen, largeScreen from "../../../css/_sizes.css";
+
+.caseStudyContainer {
```

Comment:
Shouldn't these generic case study styles be in one place and then imported?

Reply:
Yeah I definitely agree. Having looked at the other slices, they all have their own style sheets and annoyingly they have diverged a little bit so it would not be straight forward to extract generic rules for this. We should do this clean up, but maybe as a separate PR as it will touch many other files.